Gemfile

```diff
@@ -21,6 +21,8 @@ gem 'sdoc', '~> 0.4.0', group: :doc
 # Used for 'respond_to' feature
 gem 'responders', '~> 2.0'
 
+gem 'tzinfo-data'
```

Comment:
The output looks like this if I attempt to start the Rails server without `tzinfo-data`, any idea how to fix it?

![image](https://cloud.githubusercontent.com/assets/5844015/24556865/60441516-1668-11e7-9d0a-b808e7b1823f.png)